score-server/README.md

```diff
@@ -1,6 +1,6 @@
-# ICGC DCC - Storage Server
+# SCORe - Storage Server
 
-Storage server for ICGC storage system. 
+Storage server for SCORe CLI. 
```

Comment:
Not CLI in this case.

Honestly in this case I don't know what description to put here, SCORe Storage Server is the entire set of information lol. Let's just remove this description line for now.